src/Words.y

```diff
@@ -3,7 +3,7 @@
  * All rights reserved.                                                    *
  *                                                                         *
  * This is a modified version of the example from                          *
- *   http://www.lincom-asg.com/~rjamison/byacc/                            *
+ *   http://www.lincom_asg.com/~rjamison/byacc/                            *
```

Comment:
I don't think we really need this copyright any more... we're a very different file from the lex/yacc calc.